examples/creatives/base_simid_creative.js

```diff
@@ -172,4 +173,9 @@ class BaseSimidCreative {
   onVolumeChange(data) {
     this.videoState.volume = data.args.volume;
   }
+
+  onReceivePlayerLog(incomingMessage) {
+    const logMessage = incomingMessage.args['message']
```

Comment:
looks like the other functions above are using `data` as the param name, and then `data.args.message`, probably best to match that style

Reply:
that's true, in other files messages like this used incomingMessage so I based it off of that

Reply:
if you look at the simid_player.js file whenever a message is detected they use incomingMessage as the param